src/pages/admin/admin-newsfeed/admin-newsfeed.tsx

```diff
@@ -0,0 +1,80 @@
+import { Card } from "components/card/card";
+import { Checkbox } from "components/form-components/checkbox/checkbox";
+import { Input } from "components/form-components/input/input";
+import { Grid } from "components/grid/grid";
+import { useState } from "react";
+import { FormProvider } from "react-hook-form";
+
+export const AdminNewsfeed = () => {
+  const [title, setTitle] = useState("");
+  const [text, setText] = useState("");
+  
+  return (
+    <div className="flex items-center justify-center ">
+      <div className="my-profile-page-container-without-grid-template ">
+        <div className="my-profile-side-bar debug">
+          <div className="grid grid-rows-12">
```

Comment:
Make the div that contains the labels, input fields and the button (the blue strip) as big as the screen with a margin of about 24px so we use all the page, otherwise the page looks a bit empty.